test/integration/hubtemplates_variables_test.go

```diff
@@ -36,11 +37,15 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test .ManagedClusterLabels in hub
 			common.DoRootComplianceTest(policyName, policiesv1.Compliant)
 		})
 
-		It("Checks that the configmap was created correctly on the managed cluster", func() {
-			value, err := common.OcManaged("get", "configmap", createdConfigmapName, "-n=default",
-				"-o=jsonpath={.data.testvalue}")
+		It("Checks that the configmap was created correctly on the managed cluster", func(ctx SpecContext) {
+			cm, err := clientHub.CoreV1().ConfigMaps("default").Get(ctx, createdConfigmapName, metav1.GetOptions{})
```

Comment:
clientHub -> managedcluster ?

Reply:
I agree--while the Hub is technically a managed cluster, I think this may be a more intriguing test to check the managed cluster with `clientManaged`?

Reply:
...or, better, both clients? 😄